stories/3-group-by-country.md

```diff
@@ -1,25 +0,0 @@
-# 3) Group by Country
```

Comment:
was this rm'ed due to time?

Reply:
I just renamed it, could move it to a 4th task if splitting #2.